content/webapp/contexts/ConceptPageContext/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { createContext, useContext } from 'react';
+
+import { ConceptConfig } from './concept.config';
+
+type Props = {
+  config?: ConceptConfig;
+};
+
+const ConceptPageContext = createContext<Props>({
+  config: undefined,
```

Comment:
wondering; would it be nicer to have the contexts of config directly in the context? (maybe it's too faffy so consider this a very aesthetic comment)